src/lib/backends/core-lightning/index.ts

```diff
@@ -414,6 +426,103 @@ class CoreLn {
 
     return (result as ListInvoiceRequestsResponse).invoicerequests
   }
+
+  async listNodes(id: string): Promise<ListNodesResponse['nodes']> {
+    const result = (await this.connection.commando({
+      method: 'listnodes',
+      rune: this.rune,
+      params: { id }
+    })) as ListNodesResponse
+
+    return (result as ListNodesResponse).nodes
+  }
```

Comment:
not used right now - but decided to leave in place as may support work on "peers" type work